### PR TITLE
Extra data attributes for telegram.notify

### PR DESCRIPTION
**Pull request by zvldz**
## Proposed change

Added extra data attributes `parse_mode`, `disable_notification` and `message_tag` to telegram.notify.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/63604
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by zvldz** [comment]
I'm sorry, I don't understand why the tests fail.

File: source/_integrations/telegram.markdown [hunk 1]
```diff
@@ -377,3 +377,32 @@ inline_keyboard:
...
+    title: "*Send a message*"
+    message: "That's an example that sends a message with message_tag and disable_notification."
+    data:
+      message_tag: "example_tag"
+      disable_notification: True
+```
+
+{% configuration %}
+parse_mode:
+  description: Parser for the message text: `markdownv2`, `html` or `markdown`.
```
Comment (frenck) [inline source/_integrations/telegram.markdown]:
```suggestion
  description: "Parser for the message text: `markdownv2`, `html` or `markdown`."
```

File: source/_integrations/telegram_bot.markdown [hunk 2]
```diff
@@ -527,3 +527,15 @@ action:
...
 ```
+
+## Example: send_message with message tag
+
+```yaml
+action:
+- service: notify.telegrambot
+  data:
+    title: Example Message
+    message: 'Message with tag'
```
Comment (frenck) [inline source/_integrations/telegram_bot.markdown]:
```suggestion
    message: "Message with tag"
```

File: source/_integrations/telegram.markdown [hunk 3]
```diff
@@ -377,3 +377,32 @@ inline_keyboard:
...
+```
+
+{% configuration %}
+parse_mode:
+  description: "Parser for the message text: `markdownv2`, `html` or `markdown`."
+  required: false
+  type: string
+disable_notification:
+  description: True/false to send the message silently. iOS users and web users will not receive a notification. Android users will receive a notification with no sound.
+  required: false
```
Comment (frenck) [inline source/_integrations/telegram.markdown:402]:
```suggestion
  required: false
  default: false
```

**Review by frenck — APPROVED** [review]
Thanks, @zvldz 👍

**Pull request merged**